### DISCUSSION TITLE
License for the generated code?

### DISCUSSION THREAD

**Discussion by ghost** [wxFormBuilder/wxFormBuilder#655] [Q&A] [answered]
Hello and thank you developers for this program.

I was wondering of the license of the C++/Python/Lua/XRC code generated by wxFormBuilder;
Should I distribute it with GPLv2?
(I am assuming so by default as this is the main usage of the program.
Still, some code-generation programs like GCC do have different licenses for the generated code,
so I brought up an explicit question. Thanks :wink: )

Thank you!

**Comment by sodevel** [comment, marked as answer]
wxFormBuilder does create source code of the design that you create, that's basically all. It does not create anything on its own nor does it compile the source code. To my understanding the generated source code of all generators do not apply any license so you are free to apply any license you want. However the generated code uses the wxWidgets framework so at best it's limitations apply.

**Reply by ghost** [reply]
Thank you!
